pygen/pygen_src/test/riscv_instr_cov_test.py

```diff
@@ -105,9 +106,10 @@ def pre_sample(self):
                           "REMW", "REMUW"]:
             self.div_result = self.get_div_result()
 
-    def get_operand_sign(self, value):
-        #TODO: cast value to vsc.bit_t
-        if value[0]:
+    def get_operand_sign(self, operand):
+        #TODO: change operand to vsc.bit_t
```

Comment:
Is there anything blocking this TODO from being implemented in this PR?

Reply:
Nothing is blocking it actually. But as you can see, I have a bunch of these "change type" TODOs that need to be reviewed/done after completing the coverage test file (and its methods). Also some of these TODOs (those attributes belong to a `riscv_inst` object that are already defined in `riscv_instr.py` file) are dependent to the way they are defined in that file (which is not fully based on pyvsc yet).

Reply:
got it, sgtm